src/main/java/sirius/web/services/JSONCall.java

```diff
@@ -86,21 +89,30 @@ public JSONStructuredOutput getOutput() throws IOException {
      * @throws IOException in case of an IO error while receiving the result
      */
     public JSONObject getInput() throws IOException {
-        return JSON.parseObject(getPlainInput());
+        if (!outcall.isErroneous() || outcall.getHeaderField("content-type")
+                                             .toLowerCase()
+                                             .contains(MimeHelper.APPLICATION_JSON)) {
```

Comment:
`getHeaderField` is nullable! Maybe check if the header is present first, then check if it contains `json`?